GitUI/UserControls/RevisionGrid/Graph/RevisionGraphRow.cs

```diff
@@ -273,5 +274,20 @@ public void MoveLanesRight(int fromLane)
                 _segmentLanes[segment] = new Lane(lane.Index + 1, lane.Sharing);
             }
         }
+
+        /// <summary>
+        /// If this row is the parent row of <paramref name="segment"/>, returns the segment leading
+        /// to this row's first parent (if any). Otherwise, returns <paramref name="segment"/>.
+        /// </summary>
```

Comment:
```suggestion
        /// <summary>
        /// Determines the segment leading the row's first parent.
        /// </summary>
        /// <returns>
        /// The segment leading to this row's first parent (if any) if this row is the parent row of <paramref name="segment"/>;
        /// otherwise, the supplied <paramref name="segment"/>.
        /// </returns>
```